igtdetect/pdf2gloss.py

```diff
@@ -71,11 +76,17 @@ def scan_pdfs(input_path, temp_path):
                 scanned_count += 1
             except:
                 logging.error('PDF scan failed for: {}'.format(filename))
+
+            # get the doi from the pdf and save it into the dois dict
+            identifier_result = pdf2doi.pdf2doi(str(path_to_pdf))
```

Comment:
Does this also make a request to some DOI backend? or does it only read the PDF?